roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
@sdodson @ingvagabund: I still think it'd be better to ensure a minimum version of the excluder rather than doing an update each time.